tests/fxl-spread_default/OPS/package.opf

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-24</dc:date>
+    <dc:description>The default value for rendition:spread should be "auto" if not specified.</dc:description>
```

Comment:
Should also explain what happens when it's auto (copy from the auto description).